### PR TITLE
Update README #7

### PR DESCRIPTION
**Pull request by ksreenivasan**
updating the readme following github standards. added installation instructions
and details on how to contribute. Should add a contribution instructions file as
well.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,7 +7,22 @@ Coming Site Express is a small utility to create sites that indicate the user th
...
+
+
+### Prerequisities
+
+There aren't too many things to you need to set up to get Coming Site Express up and running.
+
+```
+All you need to set up is node.js and npm.
+If you're on ubuntu you should be able to do this with a simple run of
+sudo apt-get install nodejs
```
Comment (alopezm) [inline README.md:23]: In my experience, In ubuntu the users will need to install  build-essential, so they don't have problems with the dependencies of modules like semantic-release.

sudo apt-get install build-essential

File: README.md [hunk 2]
```diff
@@ -19,6 +34,21 @@ npm run generate
 
 Open in a browser **public/index.html**
 
+## Running the tests
+```
+TODO: Add an automated testing framework
+```
+
+## Contributing
```
Comment (alopezm) [inline README.md:47]: PLease add informations about the commits, we are using commitizen and sematic release, the commits should be doing using:
npm run commit

Note: The ghooks are still not properly working, but in theory in the future a user can not do commit without use this comman or follow the standars.

File: README.md (hunk 1 above)
Comment (alopezm) [reply README.md:23]: @kamikazekartik

File: README.md (hunk 2 above)
Comment (alopezm) [reply README.md:47]: @kamikazekartik

**Comment by ksreenivasan** [comment]
@alopezm : I've incorporated the changes. Can we merge?

**Pull request merged**